reana_client/cli/workflow.py

```diff
@@ -93,8 +91,16 @@ def workflow_execution_group(ctx):
 @add_access_token_options
 @click.pass_context
 def workflow_workflows(ctx, sessions, _filter, output_format, access_token,
-                       show_all, verbose):
-    """List all workflows user has."""
+                       show_all, verbose):  # noqa: D301
+    """List all user's workflows.
+
+    The `list` command returns a list of workflows or open sessions
```

Comment:
... returns the list of ...

You can also add the phrase:

By default, the list of workflows is returned. If you would like to see the list of your open interactive session, you need to pass the `--sessions` command-line option.  

because some people may not expect that we don't return the sessions by default when we say "or" in the headline.

Perhaps we should reconsider listing everything, but that is for later...